.projenrc.ts

```diff
@@ -27,6 +27,32 @@ const project = new awscdk.AwsCdkConstructLibrary({
       },
     },
   },
+  pullRequestTemplateContents: [
+    `### Issue # (if applicable)
+
+Closes #<issue number here>.
+
+### Reason for this change
+
+<!--What is the bug or use case behind this change?-->
+
+### Description of changes
+
+<!--What code changes did you make? Have you made any important design decisions?-->
+
+### Description of how you validated changes
+
+<!--Have you added any unit tests and/or integration tests?-->
+
+### Checklist
+
+- [ ] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/open-constructs/aws-cdk-library/blob/main/CONTRIBUTING.md)
+
+---
+
+_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license_
```

Comment:
Projen has a feature to enforce the contributor statement on every PR:

https://projen.io/docs/api/github?_highlight=contribu#projen.github.PullRequestLintOptions.property.contributorStatement

Reply:
I didn't know it. Thank you!